docs/getting_started/package_register.md

```diff
@@ -30,53 +30,7 @@ This guide assumes that you:
   guide to create a minimal Flyte project.
 ```
 
-## Flyte Demo Cluster
```

Comment:
May I know why this section is being removed?

Reply:
I'm moving it under the Deployment section : Sandbox. It just doesn't feel natural in this section, after discussing Tasks and Workflows, now unto Registration Patterns to suddenly go back to describing what's the sandbox cluster and also prompting the config file without much context on how it affects (if at all) the registration patterns.